src/app/hearing-part/reducers/index.ts

```diff
@@ -141,7 +142,7 @@ export const getFullHearings = createSelector(getAllHearingParts, getHearingsEnt
 export const getFullUnlistedHearings = createSelector(
     getFullHearings,
     hearings => {
-        return hearings.filter(h => !h.isListed)
+        return hearings.filter(h => h.status === 'Unlisted')
```

Comment:
it would be good to extract these statuses to some enum

Reply:
They're extracted I just forgot to use it here.